index.html

```diff
@@ -42,7 +42,159 @@ <h1>I'm Denzel Glad to see you!</h1>
+            </div>
+
+            <div class="project">
+                <div class="image_div">
+                    <img src="/images/4.png" alt="project image" class="project_image">
+                </div>
+                <div class="details_div">
+                    <h2 class="project_title">Multi-Post
+                        Stories</h2>
+                    <ul class="some_details">
+                        <li class="bold">CANPOY</li>
+                        <li class="tech"><i class="fas fa-circle"></i>Back End Dev</li>
+                        <li class="tech"><i class="fas fa-circle"></i>2015</li>
+                    </ul>
+                    <p class="project_details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="technologies_used">
+
+                        <img src="/images/Tag-1.png" alt="html-projects">
+                        <img src="/images/Tag.png" alt="css-projects">
+                        <img src="/images/Tag-2.png" alt="javaScritp-projects">
+
+                    </div>
+
+                    <div class="external_links">
+                        <button type="button" class="view_project">
+                            See Projects
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about_self">
+            <div class="Myself">
+                <h1 >About Myslef</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <p>LET’S CONNECT</p>
+            </div>
+            <div class="social_logos2">
+                <span><a href="#"><i class="fab fa-twitter"></i></a></span>
+                <span><a href="#"><i class="fab fa-linkedin-in"></i></a></span>
+                <span><a href="#"><i class="fab fa-medium-m"></i></a></span>
+                <span><a href="#"><i class="fab fa-github"></i></a></span>
+                <span><a href="#"><i class="fab fa-angellist"></i></a></span>
+            </div>
+            <div class="external_links">
+                <button type="button" class="view_project">
+                    Get my resume
+                </button>
+                <div class="coding_languages">
+                    <h3>Languages</h3>
+                    <img src="/images/Frame-9.png" alt="coding_languages">
+                    <img src="/images/Frame-10.png" alt="coding_languages">
+                    <img src="/images/Frame-11.png" alt="coding_languages">
+                    <h3>Frameworks</h3>
+                    <h3>Skills</h3>
+                </div>
+            </div>
+        </section>
     </main>
```

Comment:
This section looks quite different, it would be amazing if you can fix the details to try to replicate the Figma design you selected. Again, try to check the font-weight, images, icons, padding/margin, colors, and so on.

- [ ] Fix HTML/CSS details in all your design according to the project requirements,

![image](https://user-images.githubusercontent.com/60164239/129414357-a246df75-7a35-4c89-92f2-bb516c3bb090.png)